docs/ConfigurationFile.md

```diff
@@ -47,6 +47,26 @@ validation:
     iou_threshold: 0.4
     val_accuracy_interval: 20
 ```
+## Passing config arguments at runtime using a dict
+
+It can often be useful to pass config args directly to a model instead of editing the config file. By using a dict containing the config keys and their values. Values provided in this dict will override values provided in deepforest_config.yml.
+
+```
+# Default model has 1 class
+m = main.deepforest()
+not m.config["num_classes"] == 2
```

Comment:
What I mean here was to drop all of the equality stuff and just now `m.config["num_classes"]`. That will first show that there is 1 class and then after setting it that there are 2 classes. Basically making this make sense to the average reader instead of structuring the docs like the tests. Does that make sense?